### DISCUSSION TITLE
A possible mistake in the docs

### DISCUSSION THREAD

**Discussion by refaelsh** [nix-community/nixvim#2165] [Q&A] [answered]
Please look here: https://nix-community.github.io/nixvim/NeovimOptions/extraFiles/index.html.
The title says "extraFiles", the description talks about "extraFiles", but the example talks about vim.opts.
I thinkg its a mistake. Thoughts?

**Comment by traxys** [comment, marked as answer]
No this is expected, it's an extraFile that contains vim.opt statements

**Reply by refaelsh** [reply]
Understodod. Thank you :-)
